### PR TITLE
Add contracts, call Init

### PR DESCRIPTION
**Pull request by virgil-serbanuta**
Part of #57

### CODE REVIEW

File: mx-semantics/setup/setup.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <k> (.K => MXSetup#error) ~> MXSetup#add_account(S:String)  ... </k>
+        <mx-account-address> S </mx-account-address>
+        [priority(50)]
+    rule
+        <k> MXSetup#add_account(S:String) => .K ... </k>
+        <mx-accounts>
+            .Bag
+            =>  <mx-account>
+                    <mx-account-address> S </mx-account-address>
+                    // <mx-esdt-datas> .Bag </mx-esdt-datas>
```
Comment (ACassimiro) [inline mx-semantics/setup/setup.md:27]: Do we intend to spawn accounts with `esdt` data and storage eventually? Checking to make sure if we should keep the commented code.

File: mx-semantics/main/calls/tools.md [hunk 2]
```diff
@@ -83,6 +71,10 @@ module MX-CALLS-TOOLS
                     </mx-call-data> #as MxCallData
                 )
             => pushWorldState
+                // TODO: clearMxReturnValues is not part of the actual MX semantics,
+                // but it makes it easier to identify endpoints that do not
+                // return anything (see setVMOutput-no-retv).
+                ~> clearMxReturnValues
```
Comment (ACassimiro) [inline mx-semantics/main/calls/tools.md:77]: @virgil-serbanuta could you please elaborate a little bit more on the need to clear the return values? The reasoning is not completely clear to me.

Comment (virgil-serbanuta) [reply mx-semantics/main/calls/tools.md:91]: Thank you, the TODO text was really bad. Is it better now?

File: mx-semantics/setup/setup.md (hunk 1 above)
Comment (virgil-serbanuta) [reply mx-semantics/setup/setup.md]: I removed the comments. Sorry, I forgot about them.

File: mx-semantics/main/calls/tools.md (hunk 2 above)
Comment (ACassimiro) [reply mx-semantics/main/calls/tools.md:91]: It is much clearer now, Virgil. Thank you.

**Review by ACassimiro — APPROVED** [review]

**Pull request merged**